pyproject.toml

```
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+  "setuptools >= 42",
+  "wheel",
+  "setuptools_scm[toml]>=3.4"
+]
```

```suggestion
requires = [
  "setuptools >= 45",
  "wheel",
  "setuptools_scm>=6.2"
]
```
setuptools_scm has gotten an update to not need the toml extra anymore but relies on the tomli package directly.

---

Awesome! I was a bit unsure about these bits. First project I create with PEP517 in mind!